src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,14 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article
+    className={todo.completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}
```

Comment:
consider to use [classnames](https://www.npmjs.com/package/classnames) library (already exists in project)
for setting dynamic classnames

```
import cn from 'classnames'
```

```suggestion
    className={cn('TodoInfo', {  'TodoInfo--completed': todo.completed })}
```